__tests__/app.test.js

```diff
@@ -247,5 +247,82 @@ describe("/api/articles", () => {
           expect(body.msg).toBe("username not found");
         });
     });
+    test("PATCH: 200 - increments the votes on the specified article", () => {
+      const voteUpdate = {
+        inc_votes: 1,
+      };
+
+      return request(app)
+        .patch("/api/articles/1")
+        .send(voteUpdate)
+        .expect(200)
+        .then((response) => {
+          const { updatedArticle } = response.body;
+          expect(updatedArticle).toMatchObject({
+            article_id: 1,
+            votes: 101,
+            title: "Living in the shadow of a great man",
+            body: "I find this existence challenging",
+            author: "butter_bridge",
+            created_at: "2020-07-09T20:11:00.000Z",
+          });
+          expect(updatedArticle.votes).toBe(100 + 1);
```

Comment:
Could've just put this as 101 unless your argument is that it's for readability in which case, fair enough